### PR TITLE
Laboratory work #4, Buntova Anastasia - 20FPL1

### PR DESCRIPTION
**Pull request by DangerousCrocodile**

### CODE REVIEW

**Comment by WhiteJaeger** [comment]
It is time to push changes for the fourth lab.

**Comment by WhiteJaeger** [comment]
It is really time to push changes for the fourth lab.

File: lab_4/likelihood_text_generator_test.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def test_ideal(self):
         text_generator = LikelihoodBasedTextGenerator(profile)
         actual = text_generator.generate_sentence((4,), 4)
         expected = ((4, 5, 1), (1, 2, 1), (1, 2, 1), (1, 2, 1))
-        self.assertEqual(actual, expected)
```
Comment (WhiteJaeger) [inline lab_4/likelihood_text_generator_test.py:38]: Why are these changes needed?

**Review by WhiteJaeger — CHANGES_REQUESTED** [review]
Please discard changes in tests or provide justification.

File: lab_4/likelihood_text_generator_test.py (hunk 1 above)
Comment (DangerousCrocodile) [reply lab_4/likelihood_text_generator_test.py:38]: I don't know, I'm tired and understand nothing...
